docs/playback_filters.md

```diff
@@ -110,94 +160,122 @@ Several options from FFmpeg's readeia608 filter are accessible:
+![Broadcast Illegal Focus](media/playbackfilter_broadcast_illegal_focus.jpg)
 
-## Temporal Outlier Pixels
+## [Temporal Outlier Pixels] {#temporal-outlier-pixels}
 
 This is the same presentation as 'Normal' except that pixels that are labelled as temporal outliers are highlighted as white. Temporal outliers are pixels that significantly differ from their neighbors and often correspond to tape damage or playback error. Select **'Field'** to see the fields displayed separately.
+![Temporal Outlier Pixels](media/playbackfilter_temporal_outlier_pixels_1.jpg)
 
-## Vertical Line Repetitions
+## [Vertical Line Repetitions] {#vertical-line-repeititons}
 
 This filter displays repetitive lines of video data.
+![Vertical Line Repetitions](media/playbackfilter_vertical_line_repetitions_2.jpg)
 
-## Frame Tiles
+## [Frame Tiles] {#frame-tiles}
 
 Displays a user-defined "tiled" mosaic of successive frames. Maximum 12x12 grid. Note that because this filter requires more than one frame to display the filter will need to be in playback mode to reveal an image.
+![Frame Tiles](media/playbackfilter_frame_tiles.jpg)
 
-## Zoom
+## [Zoom] {#zoom}
 
 Allows a user to zoom to a particular portion of the image using X and Y coordinates. Includes "Strength" and "Intensity" modes.
 
-## Audio Spectrum
+![Zoom](media/playbackfilter_zoom.jpg)
+
+## [Audio Spectrum] {#audio-spectrum}
 
-Displays an visualization of the audio spectrum. Note that because this filter requires more than one frame to display the filter will need to be in playback mode to reveal an image.
+Displays a visualization of the audio spectrum. Note that because this filter requires more than one frame to display the filter will need to be in playback mode to reveal an image.
+![Audio Spectrum](media/playbackfilter_audio_spectrum.jpg)
 
-## Audio Waveform
+## [Audio Waveform] {#audio-waveform}
 
-Displays an visualization of the audio waveform. Note that because this filter requires more than one frame to display the filter will need to be in playback mode to reveal an image.
+Displays a visualization of the audio waveform. Note that because this filter requires more than one frame to display the filter will need to be in playback mode to reveal an image.
```

Comment:
Re: `Note that because this filter requires more than one frame to display the filter will need to be in playback mode to reveal an image.` This might actually be true for all of the audio playback filters. Could consider moving to a higher-level 'audio player` subsection. Could be done in a later PR as well.